README.md

```diff
@@ -9,7 +9,7 @@ Idris 2
 [![](https://github.com/idris-lang/Idris2/workflows/API/badge.svg)](https://github.com/idris-lang/Idris2/actions?query=workflow%3A"API")
 
 [Idris 2](https://idris-lang.org/) is a purely functional programming language
-with first class types.
+with first-class types.
```

Comment:
Not an error.
OED lists the first-class adj as "usually with a hyphen", and gives a couple of examples where it is without hyphen.
The codebase currently doesn't use hyphens at all, so it's a stylistic decision we can make.
I don't have a preference.